api/bibliotheca.py

```diff
@@ -1347,16 +1355,29 @@ def catch_up_from(self, start, cutoff, progress):
         added_books = 0
         i = 0
         one_day = timedelta(days=1)
+        timestamp = self.timestamp()
+
+        # If the start date is more than a month ago, then we don't consider
+        # not getting events as an error.
+        one_month_ago = datetime.utcnow() - timedelta(days=30)
```

Comment:
This is a good idea, but you don't use this variable. It should override `no_events_error = True` below.